README.rst

```diff
@@ -74,7 +74,7 @@ Usage
  # Output a parsable format (for syntax checking in editors like Vim, emacs...)
  yamllint -f parsable file.yaml
 
-`Read more in the complete documentation! <https://yamllint.readthedocs.io/>`_
+`Read more in the complete documentation! <https://yamllint.readthedocs.io/>`__
```

Comment:
Why these 2 changes?

I'm not a RST expert but I often see 1 underscore, e.g. these first results on my favorite search engine:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
The linter (specifically the part of it that pulls from Sphinx) doesn't like that we have two links with the same name ("Read more in the complete documentation!"), and the workaround suggested by a [Sphinx issue about the linter complaint](https://github.com/sphinx-doc/sphinx/issues/3921) was to make it what the [RST spec calls an Anonymous Hyperlink.](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#toc-entry-37)

I'm not very familiar with RST, but the output in GitHub seems to be identical with this change.

Reply:
OK, let's make the linter happy then :)